vmdb/app/models/customization_script.rb

```diff
@@ -1,4 +1,7 @@
 class CustomizationScript < ActiveRecord::Base
   include NewWithTypeStiMixin
   belongs_to :provisioning_manager
+
+  scope :ptables, where(:type => "CustomizationScriptPtable")
+  scope :media, where(:type => "CustomizationScriptMedium")
```

Comment:
@matthewd Is `scope` at the class level or instance level?  I thought it was the former.

> But if they're going to be scopes, they should be using lambdas (forward compatible, and generally more predictable behaviour):

Yeah, I think if you don't do it with a lambda you get a deprecation warning in Rails 4.

BTW, needing the lambda is explicitly why I like the methods better.  It's like scope is a useless DSL for something already in Ruby (i.e. taking a block of code and attaching it to a name, a.k.a. defining a method).  Unless you can info on them with something like `SomeModel.scopes`?